ydb/tests/library/harness/kikimr_config.py

```diff
@@ -432,7 +432,32 @@ def __init__(
 
             self.yaml_config["feature_flags"]["enable_external_data_sources"] = True
             self.yaml_config["feature_flags"]["enable_script_execution_operations"] = True
-
+            
+        if os.getenv("YAML_CONFIG") is not None:      
+            with open("/ydb_data/" + os.getenv("YAML_CONFIG")) as fh:
+                self.additional_yaml_config = yaml.load(fh, Loader=yaml.FullLoader)
+
+            self.yaml_config = self.merge_two_dicts(self.yaml_config, self.additional_yaml_config)
+
+    def merge_two_dicts(self, data_1, data_2):
```

Comment:
Did you see other way to merge dicts? May be standard library or some languages features?